Northstar.Custom/keyvalues/playlists_v2.txt

```diff
@@ -1,644 +1,777 @@
-playlists
```

Comment:
What's up with the huge diff in this file?

spaces <-> tabs change?

Reply:
i just uploaded my own files